File: Fourth Semester/Systems-for-Design-and-Implementation/Lab04/Lab04-Java/src/main/java/domain/Proba.java
package domain;

public class Proba implements Identifiable<Integer>{
    private String stil;
    private Integer distanta;

    public Proba(String stil, Integer distanta) {
        this.stil = stil;
        this.distanta = distanta;
    }

    public String getStil() {
        return stil;
    }

    public Integer getDistanta() {
        return distanta;
    }

    public void setStil(String stil) {
        this.stil = stil;
    }

    public void setDistanta(Integer distanta) {
        this.distanta = distanta;
    }

    @Override
    public void setId(Integer integer) {

    }

    @Override
    public Integer getId() {
        return null;
    }
}

File: Fourth Semester/Systems-for-Design-and-Implementation/Lab04/Lab04-C#/Lab04/domain/Proba.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab03.domain
{
    public class Proba : Identifiable<int>
    {

        private String stil;

        private int distanta;

        public Proba(String stil, int distanta)
        {
            this.stil = this.stil;
            this.distanta = this.distanta;
        }

        public String getStil()
        {
            return this.stil;
        }

        public int getDistanta()
        {
            return this.distanta;
        }

        public void setStil(String stil)
        {
            this.stil = this.stil;
        }

        public void setDistanta(int distanta)
        {
            this.distanta = this.distanta;
        }

       

        public void setId(int id)
        {
            throw new NotImplementedException();
        }

        int Identifiable<int>.getId()
        {
            throw new NotImplementedException();
        }
    }

}
